src/pdf.ts

```diff
@@ -217,7 +217,16 @@ class PDFJSViewer extends Widget implements DocumentRegistry.IReadyWidget {
 
   private _handleClick(evt: MouseEvent): void {
     // If it is a normal click, return without doing anything.
-    if ((IS_MAC && !evt.metaKey) || (!IS_MAC && !evt.ctrlKey)) {
+    const checkKeys = (evt: MouseEvent): boolean => {
+      let shiftAccel: boolean;
+      if (IS_MAC) {
+        shiftAccel = evt.metaKey && evt.shiftKey;
```

Comment:
Why not return `evt.metaKey && evt.shiftKey`, and do away with `shiftAccel` altogether?

Reply:
oh… because I was trying to avoid more than one return statement… which i'm now remembering is more the pattern used in javascript

Reply:
I don't have strong opinions, more of a personal preference thing.